src/aspire/utils/misc.py

```diff
@@ -109,3 +111,102 @@ def sha256sum(filename):
             h.update(mv[:n])
 
     return h.hexdigest()
+
+
+def gaussian_2d(resolution, x0=0, y0=0, sigma_x=1, sigma_y=1, peak=1, dtype=np.float64):
+    """
+    Returns a 2d Gaussian in a square 2d numpy array.
+
+    Default is a centered disc of spread=peak=1.
+
+    Note for odd resolutions center is the grid point 0,
+    while for even resolutions center is first pixel (grid point 0.5).
+
+    :param resolution: The height and width of returned array (pixels)
```

Comment:
Should stick with `size` or similar terminology for these things. Resolution often means other things which can be confusing.

Reply:
changed.